docs/Getting Started.md

```diff
@@ -0,0 +1,43 @@
+# Clone Serai
+
+```
+git clone https://github.com/serai-dex/serai.git
+```
+
+# Build and Run Serai
+
+- Ubuntu 18.04 native or WSL2 w/ Docker (Other distributions with docker may also work)
+    - Windows Guide: [https://dev.to/davidkou/running-docker-in-windows-subsystem-for-linux-wsl2-1k43](https://dev.to/davidkou/running-docker-in-windows-subsystem-for-linux-wsl2-1k43)
+    - [https://docs.docker.com/engine/install/ubuntu/](https://docs.docker.com/engine/install/ubuntu/)
+- Setup Rust & Dependencies (Assuming Linux or WSL2)
+  - Install Rust  `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+  
+## Install Solidity Compiler
+```
+     sudo add-apt-repository ppa:ethereum/ethereum \
+     sudo apt-get update \
+     sudo apt-get install solc
+```

```

Comment:
I like `solc-select` for installing solidity as you can easily switch between versions, eg for solidity 0.8.16:
```
pip3 install solc-select
solc-select install 0.8.16
solc-select use 0.8.16
```